### PR TITLE
Twitter feed again 🐦

### PR DESCRIPTION
**Pull request by rock3r**
This PR deprecates the previous Twitter UI and implements it in a new Activity.

 Before | After
 ------ | ------
 twitter_before | social_after

### CODE REVIEW

File: app/src/main/java/com/connfa/social/SocialFeedActivity.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import com.twitter.sdk.android.tweetui.SearchTimeline;
+import com.twitter.sdk.android.tweetui.TimelineResult;
+import com.twitter.sdk.android.tweetui.TweetTimelineListAdapter;
+
+import timber.log.Timber;
+
+public class SocialFeedActivity extends AppCompatActivity {
+
+    private Analytics analytics;
+
```
Comment (danybony) [inline app/src/main/java/com/connfa/social/SocialFeedActivity.java:25]: You can remove this empty line

File: app/src/main/java/com/connfa/social/SocialFeedActivity.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        tweetsAdapter = new TweetTimelineListAdapter.Builder(this)
+                .setTimeline(timeline)
+                .build();
+
+        tweetsList = (ListView) findViewById(R.id.list);
+        tweetsList.setAdapter(tweetsAdapter);
+
+        swipeLayout = (SwipeRefreshLayout) findViewById(R.id.swipe_refresh_container);
+
```
Comment (danybony) [inline app/src/main/java/com/connfa/social/SocialFeedActivity.java:53]: You can remove this empty line

File: app/src/main/java/com/connfa/ui/drawer/DrawerManager.java [hunk 3]
```diff
@@ -15,26 +16,30 @@
 
 public class DrawerManager {
 
-    public enum EventMode {Program, Bofs, Social, Speakers, Favorites, Location, About}
+    public enum EventMode {
```
Comment (danybony) [inline app/src/main/java/com/connfa/ui/drawer/DrawerManager.java:19]: 💯

File: dependencies.gradle [hunk 4]
```diff
@@ -2,16 +2,16 @@ ext {
...
-                    appCompat       : 'com.android.support:appcompat-v7:25.0.0',
+                    appCompat       : 'com.android.support:appcompat-v7:25.0.1',
                     crashlytics     : 'com.crashlytics.sdk.android:crashlytics:2.6.5',
                     gson            : 'com.google.code.gson:gson:2.8.0',
                     materialTabStrip: 'com.jpardogo.materialtabstrip:library:1.0.8',
                     parallaxScroll  : 'com.github.nirhart:parallaxscroll:1.0',
-                    playAnalytics   : 'com.google.android.gms:play-services-analytics:9.8.0',
-                    playMaps        : 'com.google.android.gms:play-services-maps:9.8.0',
+                    playAnalytics   : 'com.google.android.gms:play-services-analytics:10.0.0',
+                    playMaps        : 'com.google.android.gms:play-services-maps:10.0.0',
```
Comment (danybony) [inline dependencies.gradle:11]: Worth extracting this version number to a constant, so that it's always in sync between different play services libraries?

**Review by danybony — COMMENTED** [review]
Good stuff, just a couple of minor comments

File: app/src/main/java/com/connfa/social/SocialFeedActivity.java (hunk 1 above)
Comment (rock3r) [reply app/src/main/java/com/connfa/social/SocialFeedActivity.java:25]: It's there on purpose, analytics are separate semantically from view stuff

File: app/src/main/java/com/connfa/social/SocialFeedActivity.java (hunk 2 above)
Comment (rock3r) [reply app/src/main/java/com/connfa/social/SocialFeedActivity.java:53]: Could, but will do a refactoring pass in the next PR, would do it then if you don't mind

File: dependencies.gradle (hunk 4 above)
Comment (rock3r) [reply dependencies.gradle:11]: When we get to three, yeah

**Pull request merged**